navigator_engine/tests/unit_tests/test_data_loaders.py

```diff
@@ -186,3 +186,29 @@ def test_load_estimates_dataset_without_workflow_state(mock_engine, mocker):
             'data': {'completedTasks': []}
         }
     }
+
+
+def test_load_csv_from_zipped_resource(mock_engine, mocker):
```

Comment:
ideally we would have some tests that asserts an empty response is still raised if there is no archive file, or archive file is missing a csv file matching the reg ex.